Gemfile

```diff
@@ -20,7 +20,7 @@ gem 'turbolinks'
 gem 'arel', github: 'rails/arel', branch: 'master'
 gem 'mail', github: 'mikel/mail'
 
-gem 'sprockets', '~> 3.0.0.rc.1'
+gem 'sprockets', github: 'rails/sprockets', branch: 'master'
```

Comment:
Perhaps  we could add it here also, it is already present in the Gemfiles of the generated applications and I don't fully understand what gems should be present in Gemfile for the rails gem itself, so I didn't include it because no test was failing because of this.